myapp/urls.py

```diff
@@ -15,19 +15,15 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path, include
```

Comment:
github is being weird and doesn't seem to be updating the diffs with the latest master branch, could you re-make this pr?